Makefile

```diff
@@ -111,4 +117,13 @@ rundev:
 	cd scripts && ./run.sh
 
 run:
-	docker-compose -f docker/docker-compose.yml up
+ifeq ("$(MF_BROKER_TYPE)", "rabbitmq")
+	sed -i "s,file: brokers/.*.yml,file: brokers/rabbitmq.yml," docker/docker-compose.yml
+	sed -i 's#{MF_BROKER_URL}#MF_RABBITMQ_URL#' docker/docker-compose.yml
```

Comment:
This approach is not flexible enough and will work only when you first replace `${MF_BROKER_URL}` with `${MF_RABBITMQ_URL}`. If you want to run it with any other broker later, it won't work. Since you [changed](https://github.com/mainflux/mainflux/pull/1570/files#diff-423deb13b7c401b1a7f41ee91c77f722e11d2f317d6a66b546524e8a04cc8b03R289) `docker-compose.yml` to use `${MF_NATS_URL}`, it will not work at all. I prefer using `MF_BROKER_URL: ${MF_NATS_URL}` over `MF_BROKER_URL: ${MF_BROKER_URL}` in `docker-compose.yml`, but Makefile `sed` command needs an update.